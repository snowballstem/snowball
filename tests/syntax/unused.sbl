// Test unused variables, routines and groupings.
externals (stem)
integers (x unused_int)
booleans (b unused_bool)
strings (unused string)
routines (r)
groupings (g)
define g 'aeiou'
define r as (
   unset b
   try (next set b)
   $x = size
   b
   $(x > 2)
   g
)

define stem as true
