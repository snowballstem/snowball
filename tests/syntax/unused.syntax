define stem
  true
  Function end
