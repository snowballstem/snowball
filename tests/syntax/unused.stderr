syntax/unused.sbl:9: warning: routine 'r' defined but not used
syntax/unused.sbl:5: warning: string 'string' declared but not used
syntax/unused.sbl:5: warning: string 'unused' declared but not used
syntax/unused.sbl:4: warning: boolean 'unused_bool' declared but not used
syntax/unused.sbl:3: warning: integer 'unused_int' declared but not used
